src/DBAL/EnumTypeInitializer.php

```diff
@@ -13,7 +13,7 @@ final class EnumTypeInitializer
      * @param NamingStrategyInterface|null $strategy
      * @param bool $enumNameTypeMapping
      *
-     * @throws \Doctrine\DBAL\DBALException
+     * @throws \Doctrine\DBAL\Exception
```

Comment:
there is no such exception in `doctrine/dbal: ^2`, but we still allow it

Reply:
Yes there is. Starting from 2.11, that is why ^2.11

Reply:
👍 Tag sorting confused me. v2.10.2 is top in tag list for me, checked that one